clients/venice-client/src/main/java/com/linkedin/venice/fastclient/StatsAvroGenericStoreClient.java

```diff
@@ -176,11 +176,13 @@ private <R> CompletableFuture<R> recordRequestMetrics(
       } else if (requestContext instanceof MultiKeyRequestContext) {
         // MultiKeyRequestContext is the superclass for ComputeRequestContext and BatchGetRequestContext
         MultiKeyRequestContext<K, V> multiKeyRequestContext = (MultiKeyRequestContext<K, V>) requestContext;
+        clientStats.recordMultiKeyFanoutSize(multiKeyRequestContext.getFanoutSize());
         if (multiKeyRequestContext.retryContext != null
             && multiKeyRequestContext.retryContext.retryRequestContext != null) {
           clientStats.recordLongTailRetryRequest();
           clientStats
               .recordRetryRequestKeyCount(multiKeyRequestContext.retryContext.retryRequestContext.numKeysInRequest);
+          clientStats.recordMultiKeyFanoutSize(multiKeyRequestContext.retryContext.retryRequestContext.getFanoutSize());
```

Comment:
Can we track retry fanout size separately from the original request?
The retried request typically has few keys than the original request, so I would like to understand the behavior of the retry request.